lib/msf/base/simple/exploit.rb

```diff
@@ -154,6 +154,90 @@ def self.exploit_simple(oexploit, opts, &block)
     nil
   end
 
+
+  def setup_fail_detail_from_exception e
```

Comment:
This should have parens, per the style guide, please.  Also, as long as we're moving this around, might as well make this single-letter variable a little more meaningful:
```ruby
def set_fail_detail_from_exception(exception)
```